### DISCUSSION TITLE
How to disable snippet suggestions inside strings?

### DISCUSSION THREAD

**Discussion by nowaylifer** [saghen/blink.cmp#640] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/990e561b-8e4f-4c86-8df9-91f5331dba10)

**Comment by guillenotfound** [comment]
Did you figured out the way? I think this is a very convenient one to have.

**Reply by nowaylifer** [reply, marked as answer]
> Did you figured out the way? I think this is a very convenient one to have.

yes, there is recipe for it on plugin website https://cmp.saghen.dev/recipes.html#dynamically-picking-providers-by-treesitter-node-filetype
